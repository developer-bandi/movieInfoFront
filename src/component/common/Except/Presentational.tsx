import styles from './Style';

interface ExceptProps {
  text: string;
}

const ExceptPresentational = ({ text }: ExceptProps) => {
  return (
    <styles.MainBlock>
      <styles.Content data-testid={'except'}>{text}</styles.Content>
    </styles.MainBlock>
  );
};

export default ExceptPresentational;
