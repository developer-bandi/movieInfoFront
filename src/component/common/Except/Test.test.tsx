import { render, screen } from '@testing-library/react';
import Except from './Presentational';

describe('Except Presentational', () => {
  it('전체 화면 검증', () => {
    const utils = render(<Except text="test" />);
    expect(utils.container).toMatchSnapshot();
    screen.getByText('test');
  });
});
