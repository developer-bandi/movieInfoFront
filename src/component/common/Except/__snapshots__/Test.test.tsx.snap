// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Except/> Presentational Except 스냅샷 비교 1`] = `
<div>
  <div
    class="sc-bdvvtL hMdfsV"
  >
    <div
      class="sc-gsDKAQ cnlcfR"
      data-testid="except"
    >
      test
    </div>
  </div>
</div>
`;

exports[`Except Presentational 전체 화면 검증 1`] = `
<div>
  <div
    class="sc-bdvvtL hMdfsV"
  >
    <div
      class="sc-gsDKAQ cnlcfR"
      data-testid="except"
    >
      test
    </div>
  </div>
</div>
`;
